schema/fs.go

```diff
@@ -202,129 +202,348 @@ func _escFSMustString(useLocal bool, name string) string {
 
 var _escData = map[string]*_escFile{
 
+	"/.content-descriptor.json.un~": {
+		local:   ".content-descriptor.json.un~",
+		size:    1548,
+		modtime: 1485467092,
+		compressed: `
+H4sIAAAJbogA/wrLzJ0fmueS/5SBSaMjRGHmQ/EFfl9eT6850djLIrZo9xKlhtQ9qr06a//Zz2RgYFBl
+YGAIUAADpcwUJSsFpYySkoJiK339/ILUvOT8vJLEzLzUomK9/KJ0/eLkjNTcRP3M3MT0VP3cxLzMtNTi
+Et2czOIS/cS8vPySxJLM/LxiJR0GBgYZBgYGUwYGBkYGBgZmBghgRsIMEV25F1lAskwM8RegCpgYEAAk
+JQXj/P///z9YZBTQCER05R5jgYTwVwlo0MszMDAooohIw0RMG8GRhhy7DNAIBKljZRiNNDoAeKQxQqJI
+EhpFqtCoYQVl61poXDGhaWZGyqWjcUUpgIcklO0IZZfB2BFduReg5R04rkDxJIsUV1QvhE0bn68CBAAA
+///GLb+iDAYAAA==
+`,
+	},
+
+	"/.defs-image.json.swp": {
```

Comment:
> the -include did not do it.

It worked for me (although I had to add a trailing `$` to my [earlier guess][1].  I've filed the change as #538.

[1]: https://github.com/opencontainers/image-spec/pull/533#discussion_r98296739